src/main/java/vazkii/quark/world/feature/NaturalBlazesInNether.java

```diff
@@ -50,7 +50,8 @@ public void setupConfig() {
 
 	@Override
 	public void init(FMLInitializationEvent event) {
-		Biomes.HELL.getSpawnableList(EnumCreatureType.MONSTER).add(new SpawnListEntry(EntityBlaze.class, weight, min, max));
+		SpawnListEntry blazeEntry = new SpawnListEntry(EntityBlaze.class, weight, min, max);
+		BiomeDictionary.getBiomes(BiomeDictionary.Type.NETHER).forEach(biome -> biome.getSpawnableList(EnumCreatureType.MONSTER).add(blazeEntry));
```

Comment:
Is the biome dictionary guaranteed to be filled by init? I feel almost like this should be postInit to prevent it just happening to work due to mod load order.

Reply:
In my eyes, they should be filled. Modders should be registering their biomes with the associated RegistryEvent at this point, or if they haven't converted yet, then registering them with their associated ForgeRegistry during preInit. At whichever point, it would seem pointless (to me at least) to register them with the rest of the game, and not lump the BiomeDictionary in with that. So it should be true that everything *should* be registered with the game by this point.

However, since it is a trivial change and wouldn't have any repercussions that I can think of, I'd be willing to move it to postInit.